src/main/java/games/strategy/util/TimeManager.java

```diff
@@ -1,20 +1,63 @@
 package games.strategy.util;
 
-import java.text.DateFormat;
-import java.text.SimpleDateFormat;
-import java.util.Date;
-import java.util.SimpleTimeZone;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.FormatStyle;
 
 public class TimeManager {
   /**
-   * Replacement for Date.toGMTString();
+   * Replacement for {@code Date.toGMTString();}.
    *
-   * @param date The Date being returned as String
+   * @param instant The {@link Instant} being returned as String
    * @return formatted GMT Date String
    */
-  public static String getGMTString(final Date date) {
-    final DateFormat dateFormat = new SimpleDateFormat("dd MMM yyyy HH:mm:ss z");
-    dateFormat.setTimeZone(new SimpleTimeZone(0, "GMT"));
-    return dateFormat.format(date);
+  public static String getGMTString(final Instant instant) {
+    return DateTimeFormatter.ofPattern("d MMM yyyy HH:mm:ss 'GMT'").withZone(ZoneOffset.UTC).format(instant);
```

Comment:
I found the pattern in the source code of `Date#toGMTString`, I didn't want to change anything. But I will look how this String is being used later this day